.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+# This action is based on Github's default starter workflow for Python at
+# https://github.com/actions/starter-workflows/blob/master/ci/python-package.yml
+# (C) Github, MIT License
+
+name: "Python type checking"
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Should we pin this to a specific version (24.04?) so that we don't get surprised later?

Reply:
Yes please.